templates/webapi/main/dashboard_build_results.html.ep

```diff
@@ -15,3 +15,14 @@
         %= include 'main/group_builds', build_results => $build_results, group => $group, max_jobs => $max_jobs, children => undef, default_expanded => 0
     % }
 % }
+
+% if (@$results) {
+<p>
+  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{group}->{id}) } @$results); %>"
```

Comment:
Don't duplicate the code from the parent group overview here. You can create a separate template file and include it here and in the parent group overview.

Reply:
But the query is not the same, don't you think this adds complexity?

Reply:
I am thinking, that the url has to be passed from outside

Reply:
The other URL is generated via `url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children)` so only the name of the array is different. I suggest to simply pass the array as parameter when including the template.

By the way, the title also needs to be different as well (e.g. also using a parameter).

Reply:
WIP

Reply:
Done